server/middleware.go

```diff
@@ -84,7 +84,7 @@ func securityHeadersMiddleware(hdlr http.Handler) http.HandlerFunc {
 		// Ancient weak protection against reflected XSS (equivalent to CSP no unsafe-inline)
 		w.Header().Set("X-XSS-Protection", "1; mode=block")
 		// Prevent clickjacking attacks involving iframes
-		w.Header().Set("X-Frame-Options", "DENY")
+		w.Header().Set("X-Frame-Options", "SAMEORIGIN")
```

Comment:
Had to "enable" so we can access the iframe. Now only iframe with the same origin (protocol, host, port) will be rendered by the browser.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options